src/main/resources/view/ClientListCard.fxml

```diff
@@ -30,7 +30,7 @@
         </Label>
         <Label fx:id="name" styleClass="cell_big_label" text="\$first" />
       </HBox>
-      <FlowPane fx:id="tags" />
+      <FlowPane fx:id="products" />
```

Comment:
fx:id may be changed, but css files do not reflect this change, so the products are not styled as they were tags.